python/ray/_private/services.py

```diff
@@ -1136,9 +1135,15 @@ def start_log_monitor(
         f"--gcs-address={gcs_address}",
         f"--logging-rotate-bytes={max_bytes}",
         f"--logging-rotate-backup-count={backup_count}",
+        f"--logging-filename={logs_dir}/log_monitor.log",
```

Comment:
actually this line is not needed, can you create a follow-up to delete it?

Reply:
https://github.com/ray-project/ray/pull/51804